custom logout action in profile section (#812)
SAP/luigi: docs/navigation-parameters-reference.md

  - **label** overrides the text for the logout item. The default value is "Sign Out".
  - **testId** is a string where you can define your own custom `testId`. If nothing is specified, it is the node's label written as one word and lower case (e.g. `label`).
  - **icon** overrides the icon for the logout item. The default value is [SAP UI5 log icon](https://sapui5.hana.ondemand.com/test-resources/sap/m/demokit/iconExplorer/webapp/index.html#/overview/SAP-icons/?tag=logout).

>   - **customLogoutFn** defines a funtion to implemnt an own logout functionality. We only recommend to implement this function if no IDP Provider is configured. If an IDP Provider with a corresponding logout function is defined, the customLogoutFn on profile will ignored.

Reviewer: ```suggestion
  - **customLogoutFn** defines a function to implement your own logout functionality. It is recommended to only use  this function if no IDP Provider is configured. If an IDP Provider with a corresponding logout function is defined, the customLogoutFn on profile will ignored.
```

>   - **customLogoutFn** defines a funtion to implemnt an own logout functionality. We only recommend to implement this function if no IDP is configured. If an IDP with a corresponding [logout function](https://github.com/SAP/luigi/blob/master/docs/authorization-configuration.md) is defined , the customLogoutFn on profile will be ignored.

Reviewer: ```suggestion
  - **customLogoutFn** defines a function to implement your own logout functionality. It is recommended to only use this function if no IDP is configured. If an IDP with a corresponding [logout function](https://github.com/SAP/luigi/blob/master/docs/authorization-configuration.md) is defined , the customLogoutFn on profile will be ignored.
```
